fleetctl/fleetctl.go

```diff
@@ -101,6 +101,7 @@ var (
 		NoBlock       bool
 		BlockAttempts int
 		Fields        string
+		SshPort       int
```

Comment:
Per golang formatting guidelines, acronyms should be capitalized. Make this `SSHPort`.